examples/preprocessing/plot_smoothing_noise_quantile_transform.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+========================================================
+Effect of smoothing noise when using QuantileTransformer
+========================================================
+
+The parameter ``smoothing_noise`` can be used if some specific feature values
+are repeated exactly many times to the point of being predominant in the
```

Comment:
what do you mean by "exactly many times" this sounds weird/incomplete to me. You mean "appear multiple times" or so?